### ISSUE TITLE
task messaging via http on port 80

### ISSUE THREAD

**Issue by hjoliver** [cylc/cylc#969]
ref: https://github.com/cylc/cylc/issues/120#issuecomment-39208208

@m214089 has a few lines of C code to submit an http POST request via the curl library, to a webserver that runs cylc messaging commands from a CGI script.

This can be built into a model executable in order to trigger post-processing on files completed mid-run.

However, perhaps we should consider supporting this more generally (not just in-model) or at least providing the code and some instruction on how to use it - as it could be used instead of task polling at sites that won't open the Pyro ports or allow ssh back to the suite host, but which do have port 80 open on the HPC.

**Comment by hjoliver** [comment]
@aosprey - did you get a chance to test this last year?

**Comment by aosprey** [comment]
As I recall I did use the method to run a simple test suite, and the system worked well, however this was just on a single machine. I don't have access to a machine here where I could run a web server - our sys admin was reluctant to allow this without understanding the security implications. Therfore we decided to pass this on to our colleagues at BADC who are more expert on web services etc. I'm not sure if anyone followed it up though...
